src/mpi/romio/test/error.c

```diff
@@ -114,5 +116,5 @@ int main(int argc, char **argv)
     }
 
     MPI_Finalize();
-    return 0;
+    return (errs > 0);
```

Comment:
Why not `return errs;` ?

Reply:
Because `runtests` script does not check the value of exit status, I think simply returning 0 or 1 is sufficient.

Reply:
It's just a nitpick, but here is my thinking. Both are "sufficient". Given both are sufficient, `return (errs > 0) ` contains additional logic than just `return errs`. The former asserts what is an error condition and takes away the flexibility for `runtests` or other test scripts to get more information than just true or false. So that raises the question for reason. Without a good reason, I would prefer simpler code.